frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditOptionList/OptionListEditor.test.tsx

```diff
@@ -132,23 +132,14 @@ const openModal = async (user: UserEvent) => {
 };
 
 const renderOptionListEditor = ({ previewContextProps = {}, queries = {} } = {}) => {
-  return renderWithProviders(
-    <OptionListEditor
-      component={{
-        ...mockComponent,
-      }}
-    />,
-    {
-      queries: {
-        getOptionLists: jest
-          .fn()
-          .mockImplementation(() => Promise.resolve<OptionsLists>(apiResult)),
-        ...queries,
-      },
-      queryClient: createQueryClientMock(),
-      previewContextProps,
+  return renderWithProviders(<OptionListEditor optionsId={mockComponent.optionsId} />, {
```

Comment:
Should we define the optionsListIdMock in a simple const? Then we do not need to include the mockComponent at all in this file 🙈